src/octopeer-github/main/Options/Options.ts

```diff
@@ -14,6 +14,11 @@ const Options = new (class Options {
     private username: Boolean;
     private repo: Boolean;
     private file: Boolean;
+    private doNotWatchOnScreenEvents: Boolean;
+    private doNotWatchHoverEvents: Boolean;
+    private doNotWatchCommentElements: Boolean;
+    private doNotWatchKeyboardShortcutEvents: Boolean;
```

Comment:
I'm assuming these new options should still be implemented in the Options page? If so, please add a `TODO` marker.